src/bin/nydusd/main.rs

```diff
@@ -91,12 +92,12 @@ fn get_default_rlimit_nofile() -> Result<rlim> {
 }
 
 pub fn exit_event_manager() {
```

Comment:
We'd better use as few global variables as possible, how about wrapping an exit manager as a struct implementation, and putting it to FuseDaemon/VirtiofsDaemon? (This is not a blocker for this PR at all)

Reply:
> We'd better use as few global variables/functions as possible, how about wrapping an exit manager as a struct implementation, and putting it to FuseDaemon/VirtiofsDaemon? (This is not a blocker for this PR at all)

Absolutely right.  Event manager maybe removed, together with global shared memory - EXIT_MANAGER_RUN in later PR.